includes/html/graphs/graph.inc.php

```diff
@@ -24,13 +26,12 @@
 $output = (! empty($vars['output']) ? $vars['output'] : 'default');
 $from = parse_at_time($_GET['from']) ?: Config::get('time.day');
 $to = parse_at_time($_GET['to']) ?: Config::get('time.now');
-$graph_type = (isset($vars['graph_type']) ? $vars['graph_type'] : Config::get('webui.graph_type'));
-
 $period = ($to - $from);
-$base64_output = '';
 $prev_from = ($from - $period);
 
-$graphfile = Config::get('temp_dir') . '/' . strgen();
+$graph_image_type = $vars['graph_type'] ?? Config::get('webui.graph_type');
+$rrd_options = '';
+$auth = false;
```

Comment:
Row 41-49 is now dead code

Reply:
And many more :)
Debug code?

Reply:
Negative.  $auth is set in this file `require Config::get('install_dir') . "/includes/html/graphs/$type/auth.inc.php";`  yes this is old silly code.

Reply:
😐🔫